src/views/Tasks/InteractiveConnect/index.jsx

```diff
@@ -262,7 +265,7 @@ export default class InteractiveConnect extends Component {
       <Fragment>
         {isSessionReady && (
           <ErrorPanel
-            className={classes.warningPanel}
+            className={{ ...classes.warningPanel, ...classes.errorPanel }}
```

Comment:
Instead of creating an `errorPanel` class, use the existent `warningPanel`:

```jsx
warningPanel: {
  marginTop: theme.spacing.double,
  marginBottom: theme.spacing.double,
  ...theme.mixins.warningPanel,
},
```